modules/Microsoft.Web/staticSites/customDomains/readme.md

```diff
@@ -0,0 +1,38 @@
+# Web StaticSites CustomDomains `[Microsoft.Web/staticSites/customDomains]`
+
+This module deploys Web StaticSites CustomDomains.
+// TODO: Replace Resource and fill in description

```

Comment:
Please replace the placeholder. For example with `This module deploys custom domains for static sites.`